DOCS-2313: Improve notification URL for ecommerce integrations page (continued) (#1862)
MultiSafepay/docs: content/integration/self-made/manage-orders.md

The most important data element in our API is the **order**. Orders contain all information related to a single instance of products and/or services sold to a customer. 

Orders can be linked to multiple **transactions**. A transaction represents an instance of funds being transferred.

The most common operation to perform with our API is to create an order. An order can be set as `type`: direct or redirect. Depending which you select, affects which payment flow the customer experiences:

|  |  |  |
|---|---|---|
| **Direct** | The customer selects their payment method in your embedded checkout. | 
| **Redirect** | The customer is redirected first to a [MultiSafepay payment page](/payment-pages/) to select their payment method. | 

## Prerequisites

Before making any API requests you must:

- Have an [API key](/account/site-id-api-key-secure-code/)
- Choose your [environment](/api/#environments)

Your API key must be included in your request URL as a query parameter to be able to authenticate your request.  

> Whilst building your integration we recommend using the test environment: `https://testapi.multisafepay.com/v1/json`.

Reviewer: "Whilst" is a bit antiquated for most non-native speakers.

  },
  "customer": {
    "locale": "nl_NL",
    "ip_address": "123.123.123.123",
    "first_name": "John",
    "last_name": "Doe",
    "company_name": "Test Company Name",
    "address1": "Kraanspoor",
    "house_number": "39C",
    "zip_code": "1033SC",
    "city": "Amsterdam",
    "country": "NL",
    "phone": "0208500500",
    "email": "jdoe@example.com",
    "referrer": "https://example.com",
    "user_agent": "Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.2125.111 Safari/537.36"
  }
}'
```

Check that you receive a response with `success` set to `true`:
```
{
  "success": true,
  "data": {
    "order_id": "my-order-id-1",
    "payment_url": "https://example.com/?lang=nl_NL"
  }
}
```
If you receive an error in the response, see [Handling errors](/errors/handling-errors/).

Otherwise, open the `payment_url` to continue with completing the payment on a MultiSafepay payment page. 

For further details on how to test each payment method, see [Test payment details](/testing/test-payment-details/). If this is your first time, we recommend following the steps for [iDEAL](/testing/test-payment-details/#details-ideal).

## Get order details

> To familiarize yourself with what an order looks like in our system, try making a `GET` request for your [order details](/api/#get-order-details) with the `order_id` from the order you just created.

Reviewer: ```suggestion
To familiarize yourself with what an order looks like in our system, try making a [`GET` order details request](/api/#get-order-details) for the order you just created, using the `order_id`.
```

      "account_bic": "INGBNL2A",
      "account_holder_name": "Jan Jansen",
      "account_iban": "NL87ABNA0000000001",
      "account_id": 1,
      "external_transaction_id": "3749936454986553",
      "issuer_id": "0031",
      "recurring_flow": null,
      "recurring_id": "998107705729622024",
      "recurring_model": null,
      "type": "IDEAL"
    },
    "payment_methods": [
      {
        "account_bic": "INGBNL2A",
        "account_holder_name": "Jan Jansen",
        "account_iban": "NL87ABNA0000000001",
        "account_id": 1,
        "amount": 100,
        "currency": "EUR",
        "description": "Test Order Description",
        "external_transaction_id": "3749936454986553",
        "payment_description": "iDEAL",
        "status": "completed",
        "type": "IDEAL"
      }
    ],
    "reason": "",
    "reason_code": "",
    "related_transactions": null,
    "status": "completed",
    "transaction_id": 2345678,
    "var1": null,
    "var2": null,
    "var3": null
  }
}
```
The most important parameter is `status` which represents the [status of your order](/payments/multisafepay-statuses/) and shows the progression of the customer’s order with you. 

> The second most important parameter is `financial_status` which represents the [status of the transaction](/payments/multisafepay-statuses/) and shows the progression towards settling the funds in your MultiSafepay balance.

Reviewer: ```suggestion
The second most important parameter is `financial_status`, which represents the [status of the transaction](/payments/multisafepay-statuses/), i.e. the progression towards settling the funds in your MultiSafepay balance.
```

In your integration we recommend that you track the `status` parameter to understand how your order is progressing. For a detailled overview of the possible statuses, see [Status meanings](/payments/multisafepay-statuses/#status-meanings).

> In this example, the `status` is `completed`. This means that the customer has completed payment and settlement is guaranteed. The `financial_status` is also `completed`. This means that the funds have been settled in your MultiSafepay balance.

Reviewer: ```suggestion
In this example, the `status` is **Completed**. This means that the customer has completed payment and settlement is guaranteed. The `financial_status` is also **Completed**. This means that the funds have been settled in your MultiSafepay balance.
```

> Now that you know how to create an order, check its details, and know what is important to look for, it's the perfect time to introduce you to our webhook.

Reviewer: ```suggestion
Now that you know how to create an order and check its details, and what is important to look for, it's the perfect time to introduce you to our webhook.
```

Reviewer: Great tone you're using here. Accessible, friendly, but not cringey.

> So that you don't have to continually poll our server to see if there is an update to your order, we provide a webhook to send you notifications automatically. In the next section we explain what you need to do to configure this to work with your web server.

Reviewer: ```suggestion
So that you don't have to continually poll our server to see if there are updates to your orders, we provide a webhook to send you notifications automatically. In the next section we explain what you need to do to configure this to work with your web server.
```
